common/maplib/map.mjs

```diff
@@ -438,6 +453,35 @@ class _SingleGameMap {
     if (tile === null) return null;
     return this.graphicAsset.getTile(layer, tile);
   }
+
+  /**
+   * Expand Cell Set List.
+   * @param {string} cellSetName
+   * @return {list} Cell Set points - A list of cell set points.
+   */
+  expandCellSet(cellSetName) {
+    const expandList = [];
+    if (this.staticCellSet.has(cellSetName)) {
+      for (const cell in this.staticCellSet.get(cellSetName)) {
+        const w = (cell.w ?? 1);
+        const h = (cell.h ?? 1);
+        for (let i = 0; i < w; ++i) {
+          for (let j = 0; j < h; ++j) {
+            expandList.push(new MapCoord(this.gameMap, cell.x + i , cell.y + j));
+          }
+        }
+      }
+    }
+    return expandList;
+  }
```

Comment:
It is hard to understand what the function does from the function name and the comment. What does "expand" mean in this context?
By the way, I don't see the need for this function. See the comments of my review.

Reply:
If I have a spawn point area from (1,1) (1,5) (5,1) (5,5), I want to split it here to become 16 spawn points. Normally, getSpawnPoint() will only be called one time.